pkg/services/sqlstore/migrations/ualert/permissions.go

```diff
@@ -17,13 +17,14 @@ import (
 type roleType string
 
 const (
+	RoleNone   roleType = "None"
```

Comment:
I don't think we need to change this migration right?

Reply:
normally not, I didn't find a reason to not change it, just in case